app/assets/stylesheets/questions.css.scss

```diff
@@ -29,6 +29,14 @@
   margin-top: 0px;
 }
 
+.qset-id-modal {
+  display: none;
+}
+
+.qset-name-modal {
+  margin: 0;
```

Comment:
Not sure what the protocol is for CSS, but I think this attribute can be more general, and then we could use the same CSS class for other stylings.